src/app.module.ts

```diff
@@ -3,19 +3,36 @@ import { AppController } from './app.controller';
 import { AppService } from './app.service';
 import { ConfigModule } from '@nestjs/config';
 import { TypeOrmModule } from '@nestjs/typeorm';
+import { Season } from './entity/seasons/season.entity';
+import { ApiModule } from './entity/api/api.module';
+import { SeasonModule } from './entity/seasons/seasons.module';
+import { RouterModule } from '@nestjs/core';
 
 @Module({
   imports: [
     ConfigModule.forRoot(),
+    RouterModule.register([
+      {
+        path: 'api',
+        module: ApiModule,
+        children: [
+          {
+            path: 'seasons',
+            module: SeasonModule
+          }
+        ]
+      }
```

Comment:
note: this is fine for now, but as this list of children grows we'll likely wanna organize this in another file and import it to make the app.module less cluttered.